src/pages/components/timeline/TimelinePoint.tsx

```diff
@@ -25,9 +25,9 @@ export const pointTypeToColor: Record<PointType, string> = {
 
 export const pointTypeToDescription: Record<PointType, string | null> = {
   [PointType.BOUNDARY]: null,
-  [PointType.GTFS]: TEXTS.timestamp_gtfs,
-  [PointType.SIRI]: TEXTS.timestamp_siri,
-  [PointType.TARGET]: TEXTS.timestamp_target,
+  [PointType.GTFS]: t('timestamp_gtfs'),
+  [PointType.SIRI]: t('timestamp_siri'),
+  [PointType.TARGET]: t('timestamp_target'),
```

Comment:
```suggestion
  [PointType.GTFS]: 'timestamp_gtfs',
  [PointType.SIRI]: 'timestamp_siri',
  [PointType.TARGET]: 'timestamp_target',
```

Reply:
Here I think we need the translation modal, this is the result if we remove it from the first two:
<img width="468" alt="image" src="https://github.com/hasadna/open-bus-map-search/assets/45571546/a29196cd-f4ed-456f-8c72-52ecb7de7948">